### ISSUE TITLE
Completion for methods within a cref do not include enough info to disambiguate

### ISSUE THREAD

**Issue by davkean** [dotnet/roslyn#43139]
**Version Used**: 

**Steps to Reproduce**:

If I complete this method:

![image](https://user-images.githubusercontent.com/1103906/78632174-a934b580-78e1-11ea-944e-85cfc5fc241c.png):

``` XML 
        /// <remarks>
        /// Callers are responsible for calling <see cref="VsReferenceManagerUser.Dispose"/> on
        /// the returned instance when finished with it.
        /// </remarks>
```

I get this warning:

```
Error	CS0419	Ambiguous reference in cref attribute: 'VsReferenceManagerUser.Dispose'. Assuming 'VsReferenceManagerUser.Dispose()', but could have also matched other overloads including 'VsReferenceManagerUser.Dispose(bool)'.	Microsoft.VisualStudio.ProjectSystem.VS.Implementation	C:\CPS\src\Microsoft.VisualStudio.ProjectSystem.VS.Implementation\References\VsReferenceManagerUserFactory.cs	48	Active

```

The completion should automatically include the parentheses to disambiguate:

``` XML 
        /// <remarks>
        /// Callers are responsible for calling <see cref="VsReferenceManagerUser.Dispose()"/> on
        /// the returned instance when finished with it.
        /// </remarks>
```

**Comment by sharwell** [comment]
Related to https://github.com/dotnet/csharplang/issues/320
